big_tests/tests/service_mongoose_system_stats_SUITE.erl

```diff
@@ -101,6 +110,13 @@ user_stats_are_reported_to_google_analytics_when_mim_starts(_Config) ->
     all_event_have_the_same_client_id(),
     ok.
```

Comment:
Reduntant ok, the previous function will already return something fine. Common tests will only fail if there's an exception thrown. Same below in the other tests.